src/js/components/tasks/capture/message.tsx

```diff
@@ -0,0 +1,106 @@
+import Accordion from '@salesforce/design-system-react/components/accordion';
+import AccordionPanel from '@salesforce/design-system-react/components/accordion/panel';
+import Icon from '@salesforce/design-system-react/components/icon';
+import Textarea from '@salesforce/design-system-react/components/textarea';
+import i18n from 'i18next';
+import React, { useState } from 'react';
+
+import {
+  BooleanObject,
+  CommitData,
+  ModalCard,
+} from '@/components/tasks/capture';
+import { UseFormProps } from '@/components/utils/useForm';
+
+interface Props {
+  inputs: CommitData;
+  errors: UseFormProps['errors'];
+  handleInputChange: UseFormProps['handleInputChange'];
+}
+
+const CommitMessageForm = ({ inputs, errors, handleInputChange }: Props) => {
+  const [expandedPanels, setExpandedPanels] = useState<BooleanObject>({});
+
+  const handlePanelToggle = (groupName: string) => {
+    setExpandedPanels({
+      ...expandedPanels,
+      [groupName]: !expandedPanels[groupName],
+    });
+  };
+
+  return (
+    <form className="slds-form slds-p-around_large">
+      <ModalCard>
+        <pre>
```

Comment:
same, unless there's a particular reason it needs to be a fully pre-formatted block?